internal/sql/repository/GlobalCMCSRepository.go

```diff
@@ -21,6 +24,12 @@ const (
 	VOLUME_CONFIG      = "volume"
 )
 
+const (
```

Comment:
if we have similar type of constants plz use them or put them in common

Reply:
I am unable to find similar constant. Constant with value "CI_PIPELINE" but constant with value "CI" doesn't exist.